packages/linter/src/utils/versions.ts

```diff
@@ -1,7 +1,4 @@
-export const nxVersion = require('../../package.json').version;
```

Comment:
Why remove this?

Reply:
I thought it makes sense to have `nx` as a central source of truth when it comes to `nxVersion`.

I'll revert that.